internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider.go

```diff
@@ -200,7 +200,7 @@ func DataSource() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"id": {
+			"idp_id": {
```

Comment:
wouldn't this be a breaking change? I know it's in the data source, but what if customers are relying with some script to that field? @AgustinBettati @oarbusi

Reply:
I added this in the previous [PR](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1808/files#diff-3b955d8e92a25faa6dd8d3e000e684fe134f3484171d07860837cc78ae724bfaR204) thinking that `id` name could be used, but now I know that `id` is reserved for the Terraform state id. So `id` attribute was there all along(and still is with this change). This is why I am renaming it to be more explicit with the name and not use `id` which already exists. So it's not a breaking change.
Sorry for the confusion.